Gemfile

```diff
@@ -94,6 +94,7 @@ group :development do
 
   gem 'mina'
   gem 'mina-multistage', require: false
+  gem 'bullet'
```

Comment:
I think it would be useful if you and @equivalentideas can have a look and test it. It might be useful for you guys. If you don't find useful I can delete it, but I think it's worth to have you both testing it before that.